diff --git a/node_modules/@sveltestrap/sveltestrap/dist/Offcanvas/Offcanvas.d.ts b/node_modules/@sveltestrap/sveltestrap/dist/Offcanvas/Offcanvas.d.ts
index 7ec3af7..afad295 100644
--- a/node_modules/@sveltestrap/sveltestrap/dist/Offcanvas/Offcanvas.d.ts
+++ b/node_modules/@sveltestrap/sveltestrap/dist/Offcanvas/Offcanvas.d.ts
@@ -9,7 +9,7 @@ export interface OffcanvasProps extends HTMLAttributes<HTMLDivElement> {
   fade?: boolean;
   header?: string;
   isOpen: boolean;
-  keyboard: boolean;
+  keyboard?: boolean;
   placement?: Placement;
   scroll?: boolean;
   sm?: boolean;
